build-coredistools.sh

```diff
@@ -47,13 +47,13 @@ fi
 
 echo "Using C compiler: $C_COMPILER"
 echo "Using C++ compiler: $CXX_COMPILER"
+echo "Using llvm-tablegen: $TblGenTool"
 
 case "$TargetOSArchitecture" in
     linux-arm)
         CMakeCrossCompiling=ON
         LLVMDefaultTargetTriple=thumbv7-linux-gnueabihf
         LLVMHostTriple=arm-linux-gnueabihf
-        LLVMTargetsToBuild="ARM"
```

Comment:
Why remove this? The idea here was that linux/arm hosted build probably didn't need the ability to disassemble arm64/x86/x64/LA64/RISCV in our usage. I suppose it doesn't matter if it can, except that make the binary larger, etc.

Reply:
That was inadvertent.